### PR TITLE
Updating security group template

### PR DESCRIPTION
**Pull request by kenske**
The `security_group_id` field should not be quoted. I also fixed some of the whitespace issues.

### CODE REVIEW

File: src/terrafort/templates/aws_security_group_rule.tf [hunk 1]
```diff
@@ -1,25 +1,23 @@
-{% for range in resource.IpRanges %}
+{% for range in resource.IpRanges -%}
 {% set rule_num = count() %}
 resource "aws_security_group_rule" "{{ resource.name }}{% if rule_num > 0 %}_{{ rule_num }}{% endif %}" {
-  security_group_id = "{{ resource.id }}"
+  security_group_id = {{ resource.id }}
   type              = "{{ resource.type }}"
```
Comment (hvillanueva) [inline src/terrafort/templates/aws_security_group_rule.tf:5]: will the quotes be an issue with `resource.type` as well?

File: src/terrafort/templates/aws_security_group_rule.tf [hunk 2]
```diff
@@ -1,25 +1,23 @@
...
   to_port           = {{ resource.ToPort | default('0')}}
   protocol          = "{{ resource.IpProtocol}}"
   cidr_blocks       = ["{{ range.CidrIp }}"]
   description       = "{{ range.Description }}"
 }
-{% endfor %}
-
-{% for source in resource.UserIdGroupPairs %}
+{% endfor -%}
+{%- for source in resource.UserIdGroupPairs %}
```
Comment (hvillanueva) [inline src/terrafort/templates/aws_security_group_rule.tf:13]: no newline between these two?

**Review by hvillanueva — APPROVED** [review]

File: setup.py [hunk 3]
```diff
@@ -20,7 +20,7 @@ def read(fname):
 
 setup(
     name='terrafort',
-    version='0.2.1',
+    version='0.2.3',
```
Comment (hvillanueva) [inline setup.py:23]: why not `0.2.2`?

File: src/terrafort/templates/aws_security_group_rule.tf (hunk 1 above)
Comment (kenske) [reply src/terrafort/templates/aws_security_group_rule.tf:5]: no, this field requires quotes

File: src/terrafort/templates/aws_security_group_rule.tf (hunk 2 above)
Comment (kenske) [reply src/terrafort/templates/aws_security_group_rule.tf:13]: no, but the output will still show the newline

File: setup.py (hunk 3 above)
Comment (kenske) [reply setup.py:23]: I messed up the last release, forgot to update this for 0.2.2

**Pull request merged**